controller/controller_server/csi_controller_server.py

```diff
@@ -495,7 +495,7 @@ def get_identity_config(self, attribute_name):
         return self.cfg['identity'][attribute_name]
 
     @csi_method(error_response_type=csi_pb2.GetPluginInfoResponse)
-    def GetPluginInfo(self, _, context):
+    def GetPluginInfo(self, _, context):  # pylint: disable=invalid-name
```

Comment:
why not in the rest of the `spec` funcs?

Reply:
the linter doesn't complain.
perhaps when these ones are disabled, the class majority becomes `PascalCase`, then the linter gives up or something

Reply:
weird..